src/components/ActivityForm/ActivityForm.js

```diff
@@ -184,7 +197,11 @@ export default class ActivityForm extends React.Component {
           It will be big exercise. So, do it later (when you start real styling - do it also quite asap).
           But by separate pr only for dropdown. */}
           <Button type="submit" data-test="button-submit">
-            {this.props.activity.name ? 'Update Activity' : 'Add Activity'}
+            {this.props.activity.name ? (
+              <FormattedMessage id="activityFormBtnEdit" defaultMessage="Edit Activity" />
```

Comment:
you could make it easier. At first, destructure props and then make a conditional in id, having one `FormattedMessage`. I don't think you need defaultMessage props at all